src/main/components/RequestBar.jsx

```diff
@@ -1,8 +1,8 @@
 import React, { useState } from 'react';
 
 const RequestBar = (props) => {
-  const { SourceOrDest, requestBody, setData } = props;
-
+  const { SourceOrDest , setData, tests, setTests } = props;
```

Comment:
some small styling changes, can you remove the space after SourceOrDest? and since this is a slight longer de -structure, let make it look like this per the style guide.

const {
    SourceOrDest, setData, tests, setTests,
  } = props;

Reply:
I fixed this manually, was blocking other features